index.js

```diff
@@ -159,13 +159,19 @@ export const Rels = {
 	Outcomes: {
 		intents: 'https://outcomes.api.brightspace.com/rels/intents',
 		intent: 'https://outcomes.api.brightspace.com/rels/intent',
+		level: 'https://achievements.api.brightspace.com/rels/level',
```

Comment:
Should this not be under `achievements`? Maybe I need some background here.

Reply:
It's only used by outcomes. I don't know why it's a different domain

Reply:
Looks like the rel is being formatted incorrectly? (not going to link to it here because this is a public repo though) Either way, this should be in `Achievements` here, as the rels are organized by namespace.

This could be fixed in the LMS so that it's `outcomes` - easy enough to do since rels are an array, so you can add the new one now, then remove the old one at a later time 😄